independent-publisher-connectors/OpenAI/OpenAI.swagger.json

```diff
@@ -0,0 +1,261 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "OpenAI",
+    "description": "Connect to the OpenAI API and use the Power of GPT3, API key must be entered as \"Bearer YOUR_API_KEY\"",
+    "version": "1.0"
+  },
+  "host": "api.openai.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/v1/engines/{engine}/completions": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "id": {
+                  "type": "string",
+                  "description": "id"
+                },
+                "object": {
+                  "type": "string",
+                  "description": "object"
+                },
+                "created": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "created"
+                },
+                "model": {
+                  "type": "string",
+                  "description": "Which GPT3 Engine was used",
+                  "title": "Model",
+                  "x-ms-visibility": "internal"
+                },
+                "choices": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "text": {
+                        "type": "string",
+                        "description": "Completion text",
+                        "title": "Text"
+                      },
+                      "index": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "Number of completion",
+                        "title": "Index"
+                      },
+                      "logprobs": {
+                        "type": "string",
+                        "description": "logprobs",
+                        "title": "Logprobs"
+                      },
+                      "finish_reason": {
+                        "title": "Finish reason",
+                        "type": "string",
+                        "description": "Reason why the text finished (stop condition / natural end / length)"
+                      }
+                    }
+                  },
+                  "description": "Returned Completion(s)"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Completion",
```

Comment:
@PowerRobin Please provide a sentence instead of a word

Reply:
Changed to "GPT3 Completes your prompt"